metrics-monitoring/README.adoc

```diff
@@ -63,44 +63,48 @@ As a developer or administrator, you can view metrics in {product} to visualize
 
-* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Incoming byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:count`: Number of topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of topics is 1000.
 
-* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Outgoing byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
+--
 
-* `kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum`: Number of client connections to the instance
+Broker metrics::
++
+--
+* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data. The maximum storage in the broker is 1000 GiB.
 
-* `kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum`: Number of attempted client connections per second for the instance
+* `kafka_broker_quota_totalstorageusedbytes`: Amount of storage, in bytes, that is currently used by partitions in the broker. The storage usage depends on the number and retention configurations of the partitions. This metric must stay below the `kafka_broker_quota_softlimitbytes` metric setting.
 
-* `kubelet_volume_stats_available_bytes`: Available disk space in the instance
+* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
 
-* `kubelet_volume_stats_used_bytes`: Used disk space in the instance
+* `kafka_controller_kafkacontroller_offline_partitions_count`: Number of partitions in the cluster that are currently offline. Offline partitions cannot be used by clients for producing or consuming data. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`.
+
+* `kubelet_volume_stats_available_bytes`: Amount of disk space, in bytes, that is available in the broker.
+
+* `kubelet_volume_stats_used_bytes`: Amount of disk space, in bytes, that is currently used in the broker.
 --
 
 Topic metrics::
```

Comment:
@Mickael, I realize that you actually split the first 3 of these off as a separate "topic broker metric" section, but I'm not sure if users will know what "topic broker" metrics are. I think people more likely know what broker metrics, topic metrics, and cluster metrics are, respectively. Thought it might create more confusion splitting it that way, but let me know what you think.

Reply:
These metrics are per broker and per topic, they have both labels. I thought mentioning the labels associated with each metric was useful so I grouped them this way. If you find it confusing, you can keep them in the "topic" category.